atlasdb-cassandra/src/test/java/com/palantir/atlasdb/keyvalue/cassandra/ColumnFamilyDefinitionsTest.java

```diff
@@ -39,4 +43,20 @@ public void compactionStrategiesShouldMatchWithOrWithoutPackageName() {
                         onlyClassName.compaction_strategy),
                 ColumnFamilyDefinitions.isMatchingCf(fullyQualified, onlyClassName));
     }
+
+
+    @Test
+    public void cfDefWithOldAndNewDefaultValuesShouldNotMatch() {
```

Comment:
Maybe rename this test to `cfDefWithDifferingGcGraceSecondsValuesShouldNotMatch`. I think there's a world where we one day distinguish between different values of the cfDef being considered in the matching or not so this test should be clear that it's testing `gc_grace_seconds`